### PR TITLE
Update abstracts of litanies

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
According to the suggestions by Leonard Bahr in
https://github.com/BetaMasaheft/Documentation/issues/2553.

### CODE REVIEW

File: 3001-4000/LIT3188Liton.xml
Comment (LeonardBahr) [inline 3001-4000/LIT3188Liton.xml:1]: Looks good so far. Do we need the reference to LIT1808 two times, though? Should we delete the first mentioning?

File: 1001-2000/LIT1808Litonz.xml
Comment (LeonardBahr) [inline 1001-2000/LIT1808Litonz.xml:1]: All good. We should also change the title "Liṭon za-śark" into "Liṭon za-sark" both in the Ethiopic script and in the transcription.

**Review by LeonardBahr — CHANGES_REQUESTED** [review]

File: 3001-4000/LIT3188Liton.xml
Comment (CarstenHoffmannMarburg) [reply 3001-4000/LIT3188Liton.xml:1]: > Looks good so far. Do we need the reference to LIT1808 two times, though? Should we delete the first mentioning?

I can see LIT1808Litonz only once in this list (second place).

Comment (LeonardBahr) [reply 3001-4000/LIT3188Liton.xml:1]: But also later in

the litanies of the evening in ref type="work" corresp="LIT1808Litonz"/, including
               ref type="work" corresp="LIT7009Amlakena"/ or ref type="work" corresp="LIT7010Rassayanna"/

**Review by eu-genia — APPROVED** [review]

File: 3001-4000/LIT3188Liton.xml
Comment (CarstenHoffmannMarburg) [reply 3001-4000/LIT3188Liton.xml:1]: I formed relations with saws:formsPartOf to indicate, that LIT7009Amlakena and LIT7010Rassayanna are part of LIT1808Litonz. Is this correct?

Comment (LeonardBahr) [inline 3001-4000/LIT3188Liton.xml:1]: Hey Carsten. I only see a red area, no changed green area. What do you think about rearranging the whole section like this? (Putting in the first place the three single litanies, then the litanies of the evening with its two individual records, and finally the litanies of the weekdays with the added LIT7016Liton at the end):

            <ab>Specific litanies that can be in the collection are e.g. 
              <ref type="work" corresp="LIT1806Litonz"/>, 
              <ref type="work" corresp="LIT4713Liton"/>, 
               <ref type="work" corresp="LIT3571Litonz"/>, the litanies of the evening in <ref type="work" corresp="LIT1808Litonz"/>, 
             including
              <ref type="work" corresp="LIT7009Amlakena"/>, 
             <ref type="work" corresp="LIT7010Rassayanna"/>, the litanies of the week in <ref type="work" corresp="LIT4860Litanies"/>, 
              including 
               <ref type="work" corresp="LIT1807Litonz"/>, 
               <ref type="work" corresp="LIT4858Litany"/>, 
               <ref type="work" corresp="LIT4266Litonz"/>, 
               <ref type="work" corresp="LIT4264Salota"/>, 
               <ref type="work" corresp="LIT4268Tazakkar"/>, 
               <ref type="work" corresp="LIT4265Litonz"/>,
               <ref type="work" corresp="LIT4859Litany"/>,
               <ref type="work" corresp="LIT7016Liton"/>.
            </ab>

Comment (CarstenHoffmannMarburg) [reply 3001-4000/LIT3188Liton.xml:1]: I do not understand your request. As far as I can see, I did not delete any information in this pull request, but added some. If you wish, I will add LIT7016Liton and rearrange the list as you requested.

**Comment by eu-genia** [comment]
@LeonardBahr if you go to the "changed files" tab https://github.com/BetaMasaheft/Works/pull/1119/files#diff-9572691c91430eb6d33cbd370060bf9b38652ee9bb5a01aa3d6082a9d372f9ab you can see all changes in this branch, not just those of the latest commit, and you can expand the view to see also the unchanged code bits

@CarstenHoffmannMarburg note that due to conflicting merges there are conflicts to resolve

File: 3001-4000/LIT3188Liton.xml
Comment (LeonardBahr) [reply 3001-4000/LIT3188Liton.xml:1]: Sounds good, thank you, Carsten!

**Review by LeonardBahr — APPROVED** [review]

**Comment by eu-genia** [comment]
@LeonardBahr if what you had proposed here https://github.com/BetaMasaheft/Works/pull/1203 is already considered in this PR can you close https://github.com/BetaMasaheft/Works/pull/1203? thank you

**Comment by LeonardBahr** [comment]
@eu-genia Ok, I tried to that, hope it worked out.

**Comment by eu-genia** [comment]
should be ok

**Pull request merged**